model/manifest/manifest.go

```diff
@@ -13,11 +9,12 @@ const Collection = "manifest"
 // Branch is the branch of the repository. Modules is a map of the GitHub repository name to the
 // Module's information associated with the specific version.
 type Manifest struct {
-	Id          string             `json:"id" bson:"_id"`
-	Revision    string             `json:"revision" bson:"revision"`
-	ProjectName string             `json:"project" bson:"project"`
-	Branch      string             `json:"branch" bson:"branch"`
-	Modules     map[string]*Module `json:"modules" bson:"modules"`
+	Id              string             `json:"id" bson:"_id"`
+	Revision        string             `json:"revision" bson:"revision"`
+	ProjectName     string             `json:"project" bson:"project"`
+	Branch          string             `json:"branch" bson:"branch"`
+	Modules         map[string]*Module `json:"modules" bson:"modules"`
+	ModuleOverrides map[string]string  `json:"module_overrides,omitempty" bson:"module_overrides,omitempty"`
```

Comment:
are we ever actually putting this in the database?

Reply:
no.
I removed the extra tag.